build.cmd

```diff
@@ -19,6 +19,9 @@ IF NOT EXIST build.fsx (
 SET BuildTarget=
 if "%BuildRunner%" == "MyGet" (
   SET BuildTarget=NuGet
+
```

Comment:
Can you add a comment here in the file about what this is doing? I don't recognise the ### or * syntax (and looking up shell syntax on google is nearly impossible) and have never seen anything called "git build".